development/attributions-check.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+yarn generate:attributions
+
+ATTRIBUTIONS_FILE="./attribution.txt"
+
+# check to see if there's changes on the attributions file
+if ! git diff --exit-code "$ATTRIBUTIONS_FILE"; then
+  echo "attribution.txt has changed!"
+  echo "run \`yarn generate:attributions\` and commit changes."
```

Comment:
If I run `yarn generate:attributions`, then it creates a `licenseInfos.json` file. Should this be committed to the repo as well? Or perhaps it should be added to `.gitignore` so people don't accidentally commit it?

Reply:
Good catch, I don't think we need it I will add it to the .gitignore